roaring/roaring_test.go

```diff
@@ -191,6 +191,27 @@ func TestBitmap_Max(t *testing.T) {
 		}
 	}
 }
+func TestBitmap_BitmapCountRangeBug(t *testing.T) {
```

Comment:
let's name this test something other than `Bug`; that will become meaningless going forward.